package.json

```diff
@@ -9,18 +9,22 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@fontsource/lora": "^5.1.0",
+    "@fontsource/montserrat": "^5.1.0",
+    "@fontsource/playfair-display": "^5.1.0",
+    "@fontsource/roboto": "^5.1.0",
```

Comment:
for fonts, use `next/font`. Here's an [tutorial video](https://www.youtube.com/watch?v=L8_98i_bMMA)